message/validation/common_checks.go

```diff
@@ -86,7 +86,24 @@ func (mv *messageValidator) dutyLimit(msgID spectypes.MessageID, slot phase0.Slo
 		return 2, true
```

Comment:
@olegshmuelov @MatheusFranco99 do we need to change it to 1 because now we check `dutyCount > dutyLimit`?

Reply:
> @olegshmuelov @MatheusFranco99 do we need to change it to 1 because now we check `dutyCount > dutyLimit`?

https://github.com/ssvlabs/ssv/pull/1756#issuecomment-2368091077

Reply:
but I think the logic in this case is still different from other cases because it's max allowed plus 1 (there may be max 1 aggregator/registration duty, right?). In other cases it's just max allowed

Reply:
confirmed with @MatheusFranco99 that it should be 2, resolving the conversation